install/kubernetes/helm/istio/charts/sidecarInjectorWebhook/templates/mutatingwebhook.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: admissionregistration.k8s.io/v1beta1
 kind: MutatingWebhookConfiguration
 metadata:
-  name: istio-sidecar-injector
+  name: istio-sidecar-injector-{{ .Release.Namespace }}
```

Comment:
How will upgrade works ? I'm pretty sure it'll break.

/hold

Reply:
@costinm I would expect in an upgrade scenario coupled with an injection taking place, there might be an injection error and Kubernetes would self-heal.  Good question, and needs to be verified.